README.md

```diff
@@ -212,35 +212,35 @@ More commands will be added over time.
 
 ## Specifying Dependencies
 
-Component dependencies are interfaces defined in [wit](https://github.com/bytecodealliance/wit-bindgen)
-that are listed in a special section in the project's `Cargo.toml` file: 
+Dependencies are interfaces defined in [wit](https://github.com/bytecodealliance/wit-bindgen)
+that are listed in special sections in the project's `Cargo.toml` file.
 
-```toml
-[package.metadata.component.dependencies]
-```
+To import interfaces from your component, use the
+`[package.metadata.component.imports]` table.
 
-Dependencies are specified much like normal path dependencies in `Cargo.toml`:
+To export interfaces from your component, use the
+`[package.metadata.component.exports]` table.
 
-```toml
-binding-name = { version = "0.1.0", path = "path/to/interface.wit" }
-```
-
-By default, dependencies specified this way are for _imported_ interfaces.
-
-To specify an _exported_ interface, use the `export` key set to `true`:
+Dependencies are specified much like normal dependencies in `Cargo.toml`:
 
 ```toml
-binding-name = { version = "0.1.0", path = "path/to/interface.wit", export = true }
+binding-name = "path/to/interface.wit"
 ```
 
 To export a _default_ interface (i.e. one where the interface's functions
-are directly exported by the component itself), omit the `version` key:
+are directly exported by the component itself), set the `default` key to
+the name of the entry in the `[package.metadata.component.exports]` table.
 
 ```toml
-binding-name = { path = "path/to/interface.wit", export = true }
+[package.metadata.component]
+default = "foo"
```

Comment:
I wonder if it would be possible to consider another name like `namespace` or alternatively something more descriptive like `default-interface`? Just to avoid confusing `default` with the `"default"` export name in JS modules, where many already don't realise it's just another export name string given special syntactical sugar.

Reply:
How about `default-export` to also better tie the name to the export list?

Reply:
Although that might still be confusing. Happy with `default-interface`, either way.

Reply:
`direct-export`?

Reply:
Maybe just `interface = "foo"`? I assume this feature doesn't compose with `"exports"` anyway?

Reply:
It's intended to specify which of the exported interfaces gets its functions directly exported by the component itself rather than via an instance export.

`wit-component` currently designates a single interface for exporting directly, the rest go through instance exports, to guarantee there's never a naming conflict and you can easily "cast" a component instance to the appropriate interface at runtime without having to have some special knowledge of how to map exports to interfaces.

Reply:
Just tried it out and it all seems relatively straightforward, thanks for the clarifications. I had thought this option was a direct path itself as opposed to a key name in `"exports"`.

My main concern is just avoiding conflation of default and namespace concepts by keeping terms clear. Of course `default` can mean many things but in this context specifically it would be nice if we can be a bit careful. How about `interface-export`?

Reply:
A compromise of`direct-interface-export`? We could probably bike shed this all day 😄.

Reply:
The point was very much a comment as opposed to a change request, and it is entirely your decision if you want to keep things as-is or make a change and what name to use. Will mark as resolved for now regardless!

Reply:
Yeah, I'm definitely convinced that `default` is not conveying enough intent as to be clear, so the feedback is very appreciated.

I didn't want to dictate a name, but let's go with `direct-interface-export` for now and if that proves too verbose or unclear we can update it later.